pkg/sloop/webserver/webfiles/filter.js

```diff
@@ -75,6 +75,16 @@ function populateDropdownFromQuery(param, elementId, defaultValue, queryUrl) {
     return value
 }
 
+var selectedEndTimeValue;
+$(document).ready(function(e) {
+    $('#selectedEndTime').on('blur', function(evt) {
+        var dateTimeValue = document.getElementById('selectedEndTime').value;
+        selectedEndTimeValue = dateTimeValue.toString()
+        sessionStorage.setItem('selectedEndTime', selectedEndTimeValue);
+        sessionStorage.setItem('setSelectedEndTime', new Date().toISOString());
```

Comment:
Whats the purpose of this variable?

Reply:
this variable is basically used to keep track of the time when user pick an end time, it's used here: https://github.com/salesforce/sloop/pull/150/files#diff-0d1cf3a5e1a60113eebad58ab49d836c329c820390474fcedc837693619757c9R533,having this variable, we make sure user has a selection and we then keep users selection and render in ui

Reply:
@mengyaoyang11 why do we need `selectedEndTimeValue` var declared at the global scope, I don't see it is used anywhere except line 83, which it seems can be replaced with dateTimeValue.toString(), i.e.:
`sessionStorage.setItem('selectedEndTime', dateTimeValue.toString());`
if it is for readability/debugging - can we reduce the scope (same as `var dateTimeValue`)?

Reply:
@nurland thanks, good catch, moved selectedEndTimeValue inside the function